src/components/Section/Section.css

```diff
@@ -63,10 +67,29 @@
 }
 
 @media screen and (max-width: 768px) {
-
   .section_layout_vertical {
     flex-direction: column;
   }
+
+  .section_type_products {
+    height: 986px;
```

Comment:
This is of lower priority, but if we want it pixel perfect it'll need to be addressed. This section is rendering at 1206px, not the 986px it is supposed to be, because of the 110px padding.  Changing the height to 766px will give the correct section height, but will throw other things out of wack, so other adjustments would be necessary. 

![image](https://user-images.githubusercontent.com/37159177/112776275-37331000-900d-11eb-875b-b5dcd63d67d4.png)